packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2360,8 +2360,12 @@ export class WalletService {
       const output = opts.outputs[i];
       output.valid = false;
 
-      const addrErr = ChainService.validateAddress(wallet, output.toAddress, opts);
-      if (addrErr) return addrErr;
+      try {
+        ChainService.validateAddress(wallet, output.toAddress, opts);
+        ChainService.validateAddress(wallet, opts.from, opts);
```

Comment:
only on certain chains

Reply:
fixed to use opts third param for ETH/XRP only